pipit/readers/otf2_reader.py

```diff
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: MIT
 
+import otf2
```

Comment:
added back this import, since the tests are allowed to pass without the otf2 package being installed anyway